app/scripts/modules/core/src/pipeline/config/stages/managed/ImportDeliveryConfigExecutionDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { ExecutionDetailsSection, IExecutionDetailsSectionProps, StageFailureMessage } from 'core/pipeline';
+import { IGitTrigger } from 'core/domain';
+import { SETTINGS } from 'core/config';
+
+export function ImportDeliveryConfigExecutionDetails(props: IExecutionDetailsSectionProps) {
+  const { stage } = props;
+  const trigger = props.execution.trigger as IGitTrigger;
+  const errorDetailsAvailable = stage.isFailed && !stage.failureMessage && stage.context.error;
+  const manifestDirectory = SETTINGS.managedDelivery?.manifestBasePath + (stage.context.directory ?? '');
+  const manifestFilename = stage.context.manifest ?? SETTINGS.managedDelivery?.defaultManifest;
+  const gitRef = stage.context.ref ?? 'refs/heads/master';
```

Comment:
if there's a convenient/straightforward way to use something more human friendly here like just the branch name or commit hash, IMO that'd be preferable.

Reply:
Looks like we already do that with the summary of the trigger itself on an execution? Might be worth looking at the same fields.

Reply:
I struggled with this a bit. The stash API takes a git ref, which can be things like a commit hash or a named reference like here. To be honest I didn't try to pass in just "master" because I assumed it would fail, in which case we'd need to add some additional smarts/translation between human-friendly and git-friendly. We could certainly add that, but maybe in a separate PR?

Reply:
Isn't all this stuff available on the trigger fields themselves though (as opposed to this stage's context)? We're showing it on the pipeline executions in your screenshots (`branch: master`)

Reply:
🤦‍♂ You're totally right!

Reply:
How's this?
![image](https://user-images.githubusercontent.com/1323478/71859640-7988ce00-30a4-11ea-9144-240f6fe62e6e.png)

Reply:
ooo nice that looks really good. Only nit would be to truncate the commit hash to 7 characters (`.substring(0, 7)`)

Reply:
Done!